compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/ir/IrArrayBuilder.kt

```diff
@@ -13,6 +13,8 @@ import org.jetbrains.kotlin.ir.expressions.IrFunctionAccessExpression
 import org.jetbrains.kotlin.ir.expressions.IrGetValue
 import org.jetbrains.kotlin.ir.types.IrType
 import org.jetbrains.kotlin.ir.types.classOrNull
+import org.jetbrains.kotlin.ir.types.getArrayElementType
+import org.jetbrains.kotlin.ir.types.isBoxedArray
```

Comment:
Probably these imports aren't required

Reply:
The point of this commit (https://github.com/JetBrains/kotlin/pull/3404/commits/465e69683dad06ee5f6ebb7727f640c9df4cfc31) is to migrate `IrType` utils that convert array type to element type (and later the other commit added another util to retrieve element type from array type). These were in the same module, but then migrated to `ir.tree` so that `fir` conversion can use it (when converting `FirArrayOfCall` resolved from `arrayOf` call, which has resolved array type, yet doesn't have element type).